student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission = 'true')
```

Comment:
- [ ] For the true and false values we don't need to put the ' ' is not a string but a boolean, so the parent_permission should be `parent_permission = true` 🙃😁